components/statistic/Statistic.tsx

```diff
@@ -36,10 +38,22 @@ const Statistic: React.FC<StatisticProps & ConfigConsumerProps> = props => {
     valueRender,
     prefix,
     suffix,
+    loading,
     direction,
     onMouseEnter,
     onMouseLeave,
   } = props;
+  let skeletonProps: SkeletonProps | undefined;
+  if (typeof loading === 'boolean') {
+    skeletonProps = {
+      loading,
+    };
+  } else if (typeof loading === 'object') {
```

Comment:
I mean `loading` only accept `boolean` type. No need `SkeletonProps` type.

Reply:
but if I want to set some props for Skeleton, like `active` and `round`, `boolean` may be not enough.

Reply:
if it's unnecessary for the component，I'll fix it.

Reply:
em...something is wrong with my git. Could you close this PR, I'll create a new one.